### DISCUSSION TITLE
Differences between SST 1994m and 2003m

### DISCUSSION THREAD

**Discussion by stevosg99** [su2code/SU2#1995] [Q&A]
I just updated to V7.5.1. from V7.1.1. and I am running into issues getting the 20003m SST Model to converge for a steady state simulation. 

The issue seems to be with omega, though I cannot pinpoint exactly what is causing the error. 

I have done a validation of the ONERA wing before and was able to replicate the validation with 2003m, however, I had to lower the viscosity ratio (FREESTREAM_TURB2LAMVISCRATIO) by several orders of magnitude in order to initialize it. I was then able to increase it back to my expected value, restart,  and achieve a similar result.

The issue then, is when I attempt to replicate runs I have done of a full-size transonic business jet. The RMS Omega residual is unstable unless I make the CFL turbulence ratio (CFL_REDUCTION_TURB) exceedingly small (~0.0001) and make the viscosity ratio very small as well (~1). And at that point, the residuals don't converge at all, but stagnate near their initial values.

I have tried changing everything I can think to in the config file, and most of the sections (flow, linear solvers, numerical methods, etc.,) have no effect, with the exception of variables that affect turbulence modeling.

Any thoughts on why the same initial conditions will not produce a stable run with the 2003 solver but will with the 1994?

For the time being, I will continue to use the 1994 SST, but would prefer to utilize the 2003.

Thanks

**Comment by bigfooted** [comment]
Did you run the turbulent Onera tutorial from the website? If I run it with v2003m it runs fine.

**Comment by stevosg99** [comment]
If I simply change the turbulence model to SST 1994 or 2003, it diverges. I have to adjust the FREESTREAM_TURBULENCEINTENSITY and 
FREESTREAM_TURB2LAMVISCRATIO. Using the equations I have successfully applied in the past, that would result in the following values: 																						

% Turbulence Parameters																	
% Free-stream turbulence intensity																
FREESTREAM_TURBULENCEINTENSITY= 0.0209  												
%
% Free-stream ratio between turbulent and laminar viscosity								
FREESTREAM_TURB2LAMVISCRATIO= 2503														
%

These provide a good, converged result for the 1994 model, however the 2003 model still diverges.
For the 2003 model, changing the values, somewhat arbitrarily to the following:

% Turbulence Parameters																	 
% Free-stream turbulence intensity																
FREESTREAM_TURBULENCEINTENSITY= 0.02
%
% Free-stream ratio between turbulent and laminar viscosity								
FREESTREAM_TURB2LAMVISCRATIO= 100

Produces a decent, converged answer. Those values do produce a converged result using the 1994 model, but it does not match the experimental data as well. 
FREESTREAM_TURB2LAMVISCRATIO is the driving factor, but it is strange to me that that value would need to be different for the two models.

And reiterating what I said in the initial post, for the full scale model I am running, I cannot find a value for FREESTREAM_TURB2LAMVISCRATIO that works for the 2003 model, while my equations work for the 1994 model.

Update: It seems that the problem has to do with the stress invariant in the eddy viscosity equation

**Comment by bigfooted** [comment]
So are the freestream turbulence intensity and viscosity ratio the only 2 things that you change to get converged results? Can you share somewhere (github?) the entire case? Is the small scale model mesh different from the one in the SU2 tutorials section?

**Reply by stevosg99** [reply]
Yes, that is all I am changing.

I can share all of what I did for the Onera Wing minus the mesh itself. I made a finer (and different scale) mesh than what is in the tutorial section, but it is too large to upload to github. Is there another way I could share it?
https://github.com/stevosg99/OneraM6

I am unable to share details of the full-scale model, but the issues are very similar.

**Comment by rois1995** [comment]
Hi, I had the same problems with Omega for a test case of mine. However, I was running the v1994 version and still, I could not make it converge. I tried to non-dimensionalize the equations and it might have worked. I'll try to keep you updated. 

Any news from @stevosg99?
